chain/rust/src/plutus/utils.rs

```diff
@@ -354,52 +343,69 @@ impl CostModels {
         let mut serializer = Serializer::new_vec();
         // as canonical encodings are used, we odn't need to check the keys' bytes encodings
         // and can order this statically.
-        serializer.write_map(cbor_event::Len::Len(
```

Comment:
This might be the reason why the plutus stuff wasn't being accepted in March, I need to look into it more. This should be fixed now. (the length calculation didn't take into account v3)